test/Dialect/FIRRTL/round-trip.mlir

```diff
@@ -0,0 +1,19 @@
+// RUN: circt-opt %s | circt-opt | FileCheck %s
```

Comment:
Interesting that we apparently do not have an existing test in FIRRTL Dialect that has round trip tests?